msteams-platform/sbs-create-dashboard-widget-graph-api-call-in-Teams-toolkit.yml

```diff
@@ -0,0 +1,280 @@
+
+     1. From the Teams Toolkit left pane, select **Create a new Teams app**.
+
+        :::image type="content" source="~/assets/images/sbs-create-a-new-dashboard/create-a-new teams-app.png" alt-text="Screenshot shows the selection of create new Teams app":::
+
+     1. Select **Create a new Teams app**. 
+
+        :::image type="content" source="~/assets/images/sbs-create-a-new-dashboard/create-a-new-teams-app-2.png" alt-text="Screenshot shows the selection of new Teams app.":::
+
+     1. Select **Dashboard tab** from the Teams app section.
+
+        :::image type="content" source="~/assets/images/sbs-create-a-new-dashboard/dashboard-tab-preview.png" alt-text="Screenshot shows the selection of dashboard tab preview.":::
+
+     1. Select a programming language.
+
+        :::image type="content" source="~/assets/images/sbs-create-a-new-dashboard/javascript.png" alt-text="Screenshot shows the selection of programming language.":::
+
+     1. Select a workspace folder.
+
+        :::image type="content" source="~/assets/images/sbs-create-a-new-dashboard/default-folder.png" alt-text="Screenshot shows the selection of workspace folder.":::
+
+     1. Enter an application name.
+
+        :::image type="content" source="~/assets/images/sbs-create-a-new-dashboard/enter-application-name.png" alt-text="Screenshot shows the selection of application name.":::
+
+        The Teams tab app is created in few seconds.
+
+      In TeamsFX CLI,
+
+      * If you prefer interactive mode, execute `teamsfx` new command, then use the keyboard to go through the same flow as in Visual Studio Code.
+
+      * If you prefer non-interactive mode, enter all required parameters in one command:
+
+        `teamsfx new--interactive false--capabilities "dashboard-tab"--programming-language "TypeScript"--folder "./"--app-name dashboard-cli-001`
+
+        After you successfully created the Teams app, start local debugging via F5 in Visual Studio Code. 
+      
+        Select **Debug (Microsoft Edge)** or **Debug (Chrome)** option of your preferred browser, you'll see a tab app loaded:
+
+        :::image type="content" source="~/assets/images/sbs-create-a-new-dashboard/dashboard.png" alt-text="Screenshot shows user profile information and recent Teams activity.":::
+
+        Teams app also supports different themes, including dark and high contrast themes.
+
+        :::image type="content" source="~/assets/images/sbs-create-a-new-dashboard/dark-light-theme.png" alt-text="Screenshot shows the contrast themes.":::
+       
+   
+- title: Add a widget to the dashboard
+  durationInMinutes: 5
+  content: |
+
+     Following are the steps to add a new widget to the dashboard:
+
+     1. [Define a data model](#define-a-data-model)
+     2. [Create a data retrieve service](#create-a-data-retrieve-service)
+     3. [Create a widget file](#create-a-widget-file)
+     4. [Add the widget to the dashboard](#add-the-widget-to-the-dashboard)
+    
+     ## Define a data model
```

Comment:
There seems to be some issues with the typography of the code snippets in this section.